### ISSUE TITLE
Install artifact from another project

### ISSUE THREAD

**Issue by mceste-volume** [xvik/gradle-use-python-plugin#16]
It would be nice if it were possible to specify a file as a module to install so that we can share modules between projects in a multi module project. I’m not a seasoned Python developer therefore they may be a better way to achieve this, ideally without sharing a virtual environment in the root project.

For context, we have a `pythonBuild` task that creates a source distribution that we’d like to `pip install` in a downstream project. Another option is add the source to the `PYTHONPATH` for the downstream project.

**Comment by xvik** [comment]
>For context, we have a pythonBuild task that creates a source distribution that we’d like to pip install in a downstream project. Another option is add the source to the PYTHONPATH for the downstream project.

This should be already possible with process-specific [environment variables](https://github.com/xvik/gradle-use-python-plugin#environment-variables):

```groovy
python {
   environment 'PYTHONPATH', project.file('path to file').getAbsolutePath()
}
```

This way it would be applied for all executed python tasks (including `pipInstall`). But maybe it should be put directly to `pipInstall` task only (`pipInstall.environment 'PYTHONPATH', project.file('path to file').getAbsolutePath()`): don't know if PYTHONPATH required only for module installation. 

---

> It would be nice if it were possible to specify a file as a module to install so that we can share modules between projects in a multi module project.

I suppose you refer to installation from [local source tree](https://packaging.python.org/tutorials/installing-packages/#installing-from-a-local-src-tree).
It should be easy to support (not supported now!) it with something like:

```groovy
python.pip file('path to file')
```

which will become: `python -m pip install <path to file>`

But I need to know if `-e` (dev mode) could be ignored or it might be useful too.
Also, I will need some hello world module (local dir to use as a module) which I can use for tests (I don't use python at all, and I, of course, could read docs and find out how to write such module, but it would be super awesome if real python user could save my time).

So if you need this feature, I'm open to discussion (of how it should be) and need some help (with a real case, I can test against).

**Comment by xvik** [comment]
In case if anyone else needs local modules support, please provide a sample usage example (complete simplified use-case).

**Comment by xvik** [comment]
Released version [3.0.0](https://github.com/xvik/gradle-use-python-plugin/releases/tag/3.0.0) with [requirements file support](https://xvik.github.io/gradle-use-python-plugin/latest/guide/modules/#requirementstxt).

It should be possible now to use any modules using [native requirements syntax](https://pip.pypa.io/en/stable/reference/requirements-file-format/#requirements-file-format). 

To enable native mode:

```groovy
python.requirements.strict = false
```
(this way file would be loaded with pip: `pip -r requirements.txt`)

Requirements file, I suppose, would be like this (requirements.txt):

```
./submodule/build/something-1.0-cp34-none-win32.whl
```

**Comment by xvik** [comment]
Assuming to be solved. Please reopen if required features still missed.
